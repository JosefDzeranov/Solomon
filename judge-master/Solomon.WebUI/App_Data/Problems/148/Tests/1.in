4 3 
south south west 
south whirlpool north-west 
south-east south north 
west north-east east 
1 1
