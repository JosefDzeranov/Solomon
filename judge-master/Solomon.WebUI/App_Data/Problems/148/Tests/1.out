Whirlpool in 7 move(s)!
